src/types.ts

```diff
@@ -6,22 +6,40 @@ export interface Options {
    * Relative paths to the directory to search for components.
    * @default 'src/components'
    */
-  dirs: string | string[]
+  include_dirs: string | string[]
```

Comment:
```suggestion
  dirs: string | string[]
```
It's better to not introduce a breaking change. And I think it's ok to call it `dirs` for now.

Reply:
Have you thought about the one Feature that i didnt had time to figure out?
`exclude_dirs`, thats basically the only reason why i swapped this to `include_dirs`, other then that i agree!
Atm it would be breaking, but if the exclude feature isnt available for now, i would leave it at `dirs`!

Reply:
Yeah, maybe we can just expose `inlcudeGlobs` & `excludeGlobs` later on to it so users can have better controls on this.